### PR TITLE
feat: remove snyk integrator

### PR DESCRIPTION
**Pull request by tjsilver**
## What does this change?

Strips out the Snyk integrator which raised PRs to add a workflow to repos. 

## Why?

We are deprecating Snyk. This will allow us to simplify the logic in repocop also.

## How has it been verified?
- [x] Tested locally
- [ ] Tested on CODE (n.b. requires a dangerous deploy as we are deleting infrastructure)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,10 +82,10 @@ flowchart TB
     dev[P&E Developer]
     repocop[Repocop Lambdas]
     aws[AWS APIs]
-    
+
     snyk --> |Data from snyk populates Cloudquery tables|cq
-    github --> |Data from snyk populates Cloudquery tables|cq
-    aws --> |Data from snyk populates Cloudquery tables|cq
+    github --> |Data from dependabot populates Cloudquery tables|cq
```
Comment (adamnfish) [inline README.md:87]: 👍

File: packages/repocop/src/index.ts [hunk 2]
```diff
@@ -163,7 +163,7 @@ export async function main() {
 
 	await sendRandomRepoToSnykIntegrator(repocopRules, config, repoLanguages);
 
-	const dependencyGraphIntegratorRepoCount = 2;
+	const dependencyGraphIntegratorRepoCount = 5;
```
Comment (adamnfish) [inline packages/repocop/src/index.ts:166]: Does this mean we're also ramping up the rollout pace, here?

**Review by adamnfish — APPROVED** [review]
LGTM, thank you!

File: packages/repocop/src/index.ts (hunk 2 above)
Comment (tjsilver) [reply packages/repocop/src/index.ts:166]: Ooh that's for another PR - will remove from this one!

Comment (adamnfish) [reply packages/repocop/src/index.ts:166]: Maybe this was meant to be just in the other PR?

https://github.com/guardian/service-catalogue/pull/1329#pullrequestreview-2435554323

Comment (tjsilver) [reply packages/repocop/src/index.ts]: Yep, I've changed the base now and that's fixed it.

**Review by NovemberTang — COMMENTED** [review]
🫶

**Pull request merged**